woocommerce/woocommerce
# [CYS] Temporarily remove the survey button

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/41637
Depends on https://github.com/woocommerce/woocommerce/pull/41656


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:
Set up woocommerce-gutenberg-products-block from https://github.com/woocommerce/woocommerce-blocks/pull/11890

1. Create a new site either via local environment or JN. Make sure to install WooCommerce Beta Tester as well.
2. Enable `customize-store` feature flag in `Tools -> WCA Test Helper -> Features`
3. Go to `WooCommerce -> Home` and click on `Customize your store` task.
4. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change `status.indicator` to `major`. 
5. Refresh the page, go through the process and click `Done` in the assembler.
6. You should not see `Share feedback` button in the last step page.
<img width="681" alt="Screenshot 2023-11-29 at 15 12 56" src="https://github.com/woocommerce/woocommerce/assets/186112/745591bb-6772-4f37-b8c2-f336effa1bf7">

7. Repeat the process removing the overridden response and make sure the `Share feedback` button appears.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Hide the "Share feedback" button from the CYS preview step when AI is not available.

#### Comment 

</details>
